src/components/HeadingMain/HeadingMain.js

```diff
@@ -1,21 +1,12 @@
 import React from 'react'
 
 /**
- * Component that renders the HeadingMain component, while using the variables titel and status, which are passed down from the parent.
- * The component displays a div containing a h1 tag containing the titel variable (passed down from the parent) and uses the status variable in the
- * conditional operator to either display the status or not. Based on the isVigerend variable (which is set true if the status contains a certain value),
- * the color and border color of the status is set.
- *
- * This component is imported by the following component:
- * ContainerDetailMain
- *
- * This component is imported by the following page:
- * ContainerDetail
+ * Component that renders the HeadingMain component displaying a titel and status Vigerend or Gepubliceerd.
```

Comment:
The status can also be something else then Vigerend / Gepubliceerd.

Component that renders the HeadingMain component displaying a titel and status Vigerend or Gepubliceerd.
->
Component that renders the HeadingMain component displaying a titel and status.

Reply:
That makes sense, thanks for the explanation. And I've changed the description and removed the "Vigerend or Gepubliceerd" text.